content/en/docs/distributions/gke/deploy/oauth-setup.md

```diff
@@ -1,7 +1,7 @@
 +++
-title = "Set up OAuth for Cloud IAP"
+title = "Set up OAuth client"
```

Comment:
why the rename? it seems to me mentioning IAP is beneficial

Reply:
This is for simplify the title in `table of content`, we still explain Cloud IAP in the description. It is because `OAuth` is the core information which implies AuthN and AuthZ. We want to give to enable quick navigation by providing only core info to user.

Reply:
For discussion, I understand this might be controversial. I perceive the topic as -- we want to use Cloud IAP, therefore we need to set up an OAuth client for IAP. Sort of a proof is we follow https://cloud.google.com/iap/docs/enabling-kubernetes-howto documentation to write these scripts and it's an IAP documentation.

Therefore, in my understanding, IAP is the core information here instead of OAuth.